### ISSUE TITLE
Open link in same window does not work

### ISSUE THREAD

**Issue by eder-tuffic** [dtbaker/stylepress#25]
Hi!

I do not think anyone will see this problem but that's okay :(

The "open in the same window" function is not working. Clicking on the icon nothing happens.
![img4](https://user-images.githubusercontent.com/30321691/28639705-7a359504-721f-11e7-86dd-8d5bfe0d4e1f.jpg)

All links open in a new window.

Anybody there please?

**Comment by slidespace** [comment]
Yeah! Same here! Reeeeally annoying! Please help!

**Comment by pingram3541** [comment]
Wow, this is old and likely a simple fix.  I'll thumb it up too.

**Comment by afroniquely** [comment]
Yup! Still doesn't work for me either. Can't open links in new tab or choose to open in same window.
